docker/images/grafana_graphite/clueso-dashboard.json

```diff
@@ -1,48 +1,12 @@
 {
-  "__inputs": [
-    {
-      "name": "DS_LOCAL_GRAPHITE",
-      "label": "Local Graphite",
-      "description": "",
-      "type": "datasource",
-      "pluginId": "graphite",
-      "pluginName": "Graphite"
-    }
-  ],
-  "__requires": [
-    {
-      "type": "grafana",
-      "id": "grafana",
-      "name": "Grafana",
-      "version": "4.4.3"
-    },
-    {
-      "type": "panel",
-      "id": "graph",
-      "name": "Graph",
-      "version": ""
-    },
-    {
-      "type": "datasource",
-      "id": "graphite",
-      "name": "Graphite",
-      "version": "1.0.0"
-    },
-    {
-      "type": "panel",
-      "id": "singlestat",
-      "name": "Singlestat",
-      "version": ""
-    }
-  ],
   "annotations": {
     "list": []
   },
   "editable": true,
   "gnetId": null,
   "graphTooltip": 0,
   "hideControls": false,
-  "id": null,
+  "id": 9,
```

Comment:
what is this?

Reply:
It's like the dashboard number, it's at 9 from all the imports I did. I'll change it back to null so it can be auto set by graphana